insights/parsers/ceilometer_log.py

```diff
@@ -45,4 +59,18 @@ class CeilometerCollectorLog(LogFileOutput):
         2016-11-09 14:36:35.465 4259 INFO cotyledon [-] Caught signal (15) during service initialisation, delaying it
         2016-11-09 14:38:07.280 31638 WARNING oslo_reports.guru_meditation_report [-] Guru meditation now registers SIGUSR1 and SIGUSR2 by default for backward compatibility. SIGUSR1 will no longer be registered in a future release, so please use SIGUSR2 to generate reports.
     """
-    pass
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+                returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+                If not supplied, all available lines are searched.
+
+        Yields:
+            Log lines with time stamps after the given date and time, in the
+            same format they were supplied.
+        """
+        return super(CeilometerCollectorLog, self).get_after(timestamp, lines, '%Y-%m-%d %H:%M:%S')
```

Comment:
@PaulWay this is using the same timestamp format as the base class so why implement this here?